### DISCUSSION TITLE
Copilot for VS Code cannot be turned off

### DISCUSSION THREAD

**Discussion by jack-jjm** [community/community#55827] [Copilot Conversations]
### Select Topic Area

Bug

### Body

The "disable copilot" button in VSCode and the similar option in the Command Palette just don't work - Copilot stays turned on. Nothing in the Output pane for Copilot, it just silently doesn't do anything.

VSCode 1.78.2
Copilot 1.86.82

**Comment by adityakrcodes** [comment]
The same thing is happening with me too!

My System (If it helps to find the issue is OS related or the application)
OS: Arch Linux x86_64
Kernel: 6.3.1-arch2-1 
Shell: zsh 5.9 
DE: Plasma 5.27.5 
WM: KWin

**Comment by LinqLover** [comment]
Same issue. Neither "disable for this file type" nor "disable globally" work - the icon in the bottom right corner indicates disablement but completions keep coming. I believe that this has worked pretty recently.

**Comment by olfMombach** [comment]
I have the same issue

Version: 1.87.1 (user setup)
Commit: 1e790d77f81672c49be070e04474901747115651
Date: 2024-03-06T00:21:36.258Z
Electron: 27.3.2
ElectronBuildId: 26836302
Chromium: 118.0.5993.159
Node.js: 18.17.1
V8: 11.8.172.18-electron.0
OS: Windows_NT x64 10.0.19045
